airflow/providers/cncf/kubernetes/utils/xcom_sidecar.py

```diff
@@ -47,13 +47,15 @@ class PodDefaults:
     )
 
 
-def add_xcom_sidecar(pod: k8s.V1Pod) -> k8s.V1Pod:
+def add_xcom_sidecar(pod: k8s.V1Pod, image: str) -> k8s.V1Pod:
```

Comment:
Rather than making this argument required here, i would make it optional.  Then only override it if supplied.

probably best to do kwargs only?

Reply:
Moved to `kwargs`